### PR TITLE
SC: Resolve failed unit test (TestKLAYTransferLongRangeRecovery) 

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes

This PR resolved failed unit test (TestKLAYTransferLongRangeRecovery) related with Service Chain Value Transfer Recovery.

## Types of changes

Please put an x in the boxes related to your change.

- [x] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by KimKyungup** [comment]
Dear reviers, To remove sleep, we need to write wait group and modify all related unit test.
It's a big work. So I will do that later.

File: node/sc/vt_recovery_test.go [hunk 1]
```diff
@@ -186,43 +186,52 @@ func TestKLAYTransferLongRangeRecovery(t *testing.T) {
 			}
 		}
 	})
+	// TODO-Klaytn need to remove sleep
+	time.Sleep(1 * time.Second)
```
Comment (jeonghonah) [inline node/sc/vt_recovery_test.go]: `prepare` has a WaitGroup to sync request txs and handle txs. Do we need the `Sleep` here?

Comment (KimKyungup) [reply node/sc/vt_recovery_test.go]: Yes. After waiting for handle transaction, Commit() will be executed. Then updateRecoveryHint will update the status for checking.

**Comment by KimKyungup** [comment]
> Except some comments, LGTM.
> 
> It would be nice to find out more about the cause.

After Hot fix, in test we should not check the size of pending Event List. Because, after pushing any event, bridge info loop will handle the event in ascending nonce order, not wait next nonce.

**Comment by KimKyungup** [comment]
> > Except some comments, LGTM.
> > It would be nice to find out more about the cause.
> 

@jeonghonah  After Hot fix, in test we should not check the size of pending Event List. Because, after pushing any event, bridge info loop will handle the event in ascending nonce order, not wait next nonce.

**Pull request merged**